waterloop-site/package.json

```diff
@@ -6,6 +6,7 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "@styled-icons/evaicons-outline": "^10.34.0",
```

Comment:
Instead of installing this new dependency, let's instead use the library we currently have already installed, `@material-ui/icons`.
To import the icon, you should be able to do something like:
`import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';`

Note: We want to try to not install new dependencies unless there's a definite need for it that cannot be fulfilled by the currently installed ones.